invisible_cities/evm/histos_test.py

```diff
@@ -20,7 +20,8 @@
 from .. evm.histos  import HistoManager, Histogram
 
 
-characters = tuple(string.ascii_letters + string.digits + "-")
+characters = tuple(string.ascii_letters + string.digits)
+letters    = tuple(string.ascii_letters)
```

Comment:
No need for `tuple` here. Strings are perfectly good iterables, and `text` can consume them directly.